pkg/handler/handler_test.go

```diff
@@ -200,12 +245,25 @@ func TestBadInput(t *testing.T) {
 func p(i int32) *int32 {
 	return &i
 }
+
+// Util function to get title format
+func title(year int, utgave int) string {
+	return fmt.Sprintf("readme utgave nr. %d %d", utgave, year)
```

Comment:
More fan of having this explicit in the tests instead, since that makes it easier to read them